esmvalcore/preprocessor/_multimodel.py

```diff
@@ -609,13 +632,14 @@ def _multicube_statistics(cubes, statistics, span, ignore_scalar_coords=False):
 
     # Calculate statistics
     statistics_cubes = {}
+    lazy_input = any(cube.has_lazy_data() for cube in cubes)
```

Comment:
shouldn't be `all()`? Or you accept non-lazy cubes too

Reply:
Yes, if one or a few of the cubes is not lazy (due to fixes or temporal interpolation), it's probably still more efficient not to realize all cubes. Especially if you're doing a statistics computation over e.g. all of CMIP6. If this turns out to be a problem when testing the ESMValTool recipes for the upcoming v2.9 release, we can always change it.

Reply:
sounds good and a good call too! Maybe add a #TODO there? Not too fussy about it tho

Reply:
Let's create an issue when it actually turns out to be an issue ;-)